german/ruderboot.md

```diff
@@ -42,4 +42,4 @@ Werden Sie:
 
 [Zum fliegenden Spaghettimonster beten, damit sie ueberleben?](beten/beten.md)
 
-[Das MacGyver-Messer aus der Hosentasche zu holen und das Ruderboot zu einem U-Boot umbauen] (MacGyver/uboot.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.